cli/commands/handler/interactive.go

```diff
@@ -218,6 +229,7 @@ func (opts *handlerOpts) Copy(handler *types.Handler) {
 	handler.Organization = opts.Org
 
 	handler.Command = opts.Command
+	handler.EnvVars = strings.Split(opts.EnvVars, ",")
```

Comment:
See `cli/commands/helpers/flags.go` there is a func for handling whitespace in comma separated lists. May help if someone tries `--env-vars 'ENV=production, WOOF=PACK ,     ETC=ETCD'`